frontend/src/pages/accounts/profile.page.tsx

```diff
@@ -373,6 +423,47 @@ const Profile: NextPage = () => {
   const topBanners = allAliases.length > 0 ? banners : null;
   const bottomBanners = allAliases.length === 0 ? banners : null;
 
+  // Render the upsell banner when a user has reached the free mask limit
+  const UpsellBanner = () =>
+    freeMaskLimitReached && isPhonesAvailableInCountry(runtimeData.data) ? (
```

Comment:
Nit: Instead of a ternary operator and explicitly returning `null` we could just use `&&` which will also return `null` if the condition evaluates to `false` …

```suggestion
    freeMaskLimitReached && isPhonesAvailableInCountry(runtimeData.data) && (    
```
… and then replace `) : null;` with just `);` on `L465`.